internal/net/call/call.go

```diff
@@ -942,15 +949,27 @@ func (c *serverConnection) readRequests(ctx context.Context, hmap *HandlerMap, o
 // runHandler runs an application specified RPC handler at the server side.
 // The result (or error) from the handler is sent back to the client over c.
 func (c *serverConnection) runHandler(hmap *HandlerMap, id uint64, msg []byte) {
-	// Extract request header from front of payload.
-	if len(msg) < msgHeaderSize {
+	msgLen := uint32(len(msg))
+	hdrLenEndOffset := uint32(4)
```

Comment:
This can be a "const hdrLenEndOffset".

Also, maybe hdrLenLen would be a better name?

Reply:
Done